auth/api/tests/Admin/AdminTest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Tests\Admin;
+
+use Alchemy\AdminBundle\Tests\AbstractAdminTest;
+
+class AdminTest extends AbstractAdminTest
+{
+    /*
+     * todo EA3 : fix test that now fails cause 500 on /admin after login (?)
+     */
+    public function testAdmin()
+    {
+        // $this->doTestAllPages();
```

Comment:
Should be fixed

Reply:
rollback'ed

Reply:
this new test is copied from expose (where it runs ok)
because it did not exist before, i suggest to fix this later.